vite.config.ts

```diff
@@ -4,7 +4,7 @@ import react from '@vitejs/plugin-react';
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
-  base: './',
+  base: '/',
```

Comment:
this one should brake `npm build` why do you change it?

Reply:
there is a problem on the product page. when i go to the product page and reload the page, there is a problem, only a white screen appears. therefor i changed vite.config.ts , routes.tsx  and i created netlify.toml .